cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -274,6 +274,10 @@ export default {
     cy.expect(section.find(HTML(including('FOLIO'))).absent());
   },
 
+  checkSourceValue: (expectedValue) => {
+    cy.expect(source.has({ value: expectedValue }));
+  },
+
```

Comment:
checkout this method `checkSource` in `cypress\support\fragments\inventory\holdingsRecordView.js`

Reply:
it's different page. From TC I need to check source value on Instance page

Reply:
thanks for pointing it out, i edited the TC so it correctly checks the holdings source